src/SFA.DAS.Reservations.Functions.Reservations.AcceptanceTests/Features/CreateReservation.feature

```diff
@@ -0,0 +1,14 @@
+﻿Feature: Reservation created
+	In order to know when a reservation has been created
+	I want to be told when the reservation search index is updated
```

Comment:
Slightly wrong wording should be something more like
`I want to be notified when the reservation is created and the search index is updated`

Reply:
Done